### ISSUE TITLE
Sarah comments

### ISSUE THREAD

**Issue by wright13** [nationalparkservice/DRR_Template#23]
- I like the idea of splitting the DRR out from QC and metadata generation
    - I know that some people like using the DRR as a place for QC, EDA, etc. so I think it's worth considering how we can support this outside of the DRR
- Ideally we should be able to harvest most of this info from the data package/metadata
- Maybe include this as an Rmd template in NPSdataverse?

**Comment by joe-devivo** [comment]
Putting this here so it lives with the template, but I spoke with Rob about it briefly last week. When I created the 0.1 version of the template, it was specifically with the intent of making the QC and data processing a reproducible (and documented) process where the script generates both the data package and the DRR in one fell swoop. In practice, creating a DRR is actually three tightly interwoven--and usually iterative--components: dataset QC and finalization, DRR production, and data packaging (including metadata and manifest generation). I like separating these explicitly, but worry about separating the data prep from the DRR entirely--mostly because if you do so there's no reason to use the DRR template all. Authors can/should just use a word template to generate a DRR if the reproducible content part from the DRR is fully separated.

An option that I was going to play with on my next tinkering is to call a data processing RMD file from within your DRR script, which I think is something that could be mirrored for any reproducible reporting (trend or other anlalysis reports).  

I found the following, which may be a good starting point: https://stackoverflow.com/questions/70535350/run-rmarkdown-rmd-from-inside-another-rmd-without-creating-html-output
